pylint/checkers/base/comparison_checker.py

```diff
@@ -58,6 +58,11 @@ class ComparisonChecker(_BasicChecker):
             "comparison-with-itself",
             "Used when something is compared against itself.",
         ),
+        "R0125": (
+            "Redundant comparison - %s",
+            "comparison-of-two-literals",
```

Comment:
```suggestion
            "comparison-of-constants",
```
We can anticipate the next checks where we'll be more daring and try to also handle constants.

Reply:
You mean like `if PACKET_LEN > BUFFER_LEN:"? I think that kind of code is okay actually. Especially before you settle on a value for each. Even if they're constant over a program's lifetime does not mean they're all-time constant.